### PR TITLE
Code Snippet Updates

### PR DESCRIPTION
**Pull request by zofskeez**
This PR adds the CodeSnippet component to the ldap library accounts view and updates the styling to match the original designs. 

_Before with CodeSnippet component_
image

_After style updates to CodeSnippet component_
image

_Usage in KV_
image

### CODE REVIEW

File: ui/lib/ldap/addon/components/page/library/details/accounts.hbs [hunk 1]
```diff
@@ -1,39 +1,41 @@
-<div class="has-top-margin-l is-flex-align-start">
```
Comment (zofskeez) [inline ui/lib/ldap/addon/components/page/library/details/accounts.hbs:1]: The diff is hard to read but I just changed the classes on the top level div to grid and then wrapper the `Hds::Card::Container` in a div. Other than that it's the same.

Comment (kiannaquach) [reply ui/lib/ldap/addon/components/page/library/details/accounts.hbs:1]: yay for grids! 🙌

File: ui/lib/core/addon/components/code-snippet.hbs [hunk 2]
```diff
@@ -3,13 +3,15 @@
...
+<div
+  class="has-padding-s has-background-gray-900 border-radius-4 is-flex-between is-flex-center"
+  data-test-code-snippet
+  ...attributes
+>
+  <code class="has-text-white is-size-7">{{@codeBlock}}</code>
   {{! replace with Hds::Copy::Button }}
   <CopyButton
-    class="code-snippet-copy-button"
+    class="button is-compact is-transparent has-text-grey-light"
```
Comment (kiannaquach) [inline ui/lib/core/addon/components/code-snippet.hbs:14]: thanks for using our helper classes!!

**Review by kiannaquach — APPROVED** [review]
🎉

**Pull request merged**